masterfabric-mobile/swift-camp
# CL-Images at Swift

## 📋 PR Description

The ImageView struct is a SwiftUI View that presents an image gallery and a button. It is designed to showcase a small exhibition with images of Atatürk.

---

## ✅ Checklist


---

## 🛠 Steps to Test

1. Open the project in Xcode and ensure all dependencies are resolved.
2. Navigate to the module or screen where the Image is implemented.
3. See the big picture at view
4. Please click "1881-∞" button to go back
5. Run the application and ensure there are no layout issues or unexpected crashes.

---

## 🔗 Related Links


- Documentation: https://developer.apple.com/documentation/swiftui/image

---

### 📷 Screenshots (Optional)

<img width="412" alt="13" src="https://github.com/user-attachments/assets/744d7594-7389-442e-bea7-4e8447ace720" />
<img width="412" alt="14" src="https://github.com/user-attachments/assets/a7cf2b0b-bb7f-41e6-a223-f1467bf48d71" />
